scripts/examples/gn_silabs_example.sh

```diff
@@ -315,6 +318,18 @@ else
         optArgs+="openthread_root=\"$GSDK_ROOT/util/third_party/openthread\" "
     fi
 
+    if [ "$USE_DOCKER" == true ] && [ "$USE_WIFI" == true ]; then
+        if [ "$SILABS_WISECONNECT_SDK_VERSION" == "2" ]; then
+            echo "Switching WiseConnect 2 SDK ROOT"
+            optArgs+="wiseconnect_sdk_root=\"$WISECONNECT_SDK_ROOT\" "
```

Comment:
Should this and the same wifi_sdk be handled in the gn files directly 


You can have check for environment variables in gn without doing all this logic here

Reply:
i agree with junior, this should be in the gn files. The examples can be built without using the build script.

Reply:
Removed the logic, understood that declaring this will not cause an issue on the build system.